account_multicurrency_revaluation_type/wizard/wizard_currency_revaluation.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class WizardCurrencyRevaluationType(models.TransientModel):
+
+    _inherit = 'wizard.currency.revaluation'
+
+    # Todo get in count direct/indirect rate
+
+    @api.model
+    def _get_default_rate_type(self):
+        """
+        Get default rate type if one is defined in company settings
+        """
+        return self.env.user.company_id.rate_type
+
+    rate_type = fields.Selection(
+        string='Rate type',
+        selection=[
+            ('average', 'Average'),
+            ('daily', 'Daily'),
+        ],
+        default=lambda self: self._get_default_rate_type(),
+    )
+
+    @api.model
+    def _compute_unrealized_currency_gl(self, currency_id, balances, form):
+        """
+        Override base wizard method to inject month currency evaluation
+        """
+        if self.rate_type == 'daily':
```

Comment:
I feel like the whole override is probably not needed.
Won't it work if you just do this ?
```
    def _compute_unrealized_currency_gl(self, currency_id, balances, form):
        if self.rate_type ==  'monthly':
            self = self.with_context(monthly_rate=True)
        return super()._compute_unrealized_currency_gl(currency_id, balances, self)
```
Maybe it works only after cherry-picking https://github.com/OCA/account-closing/pull/103/commits/a32ce494d9f4f08b82ffd664d5261a8964ff34e8 ?